kyo-core/shared/src/main/scala/kyo/Channel.scala

```diff
@@ -64,6 +67,28 @@ object Channel:
                 }
             }
 
+        /** Puts elements into the channel as a batch, asynchronously blocking if necessary. Breaks batch up if it exceeds channel capacity.
+          *
+          * @param values
+          *   Chunk of elements to put
+          */
+        def putBatch(values: Seq[A])(using Frame): Unit < (Abort[Closed] & Async) =
+            if values.isEmpty then ()
+            else
+                IO.Unsafe {
+                    var i = 0
+                    boundary[Unit < (Abort[Closed] & Async)]:
```

Comment:
`boundary`/`break` is a dangerous primitive to mix with value-based effect composition since it's based on exception throwing. The use seems correct but could we avoid it?